stacks-signer/src/signerdb.rs

```diff
@@ -425,9 +504,17 @@ static SCHEMA_3: &[&str] = &[
     "INSERT INTO db_config (version) VALUES (3);",
 ];
 
+static SCHEMA_4: &[&str] = &[
+    CREATE_TENURE_BLOCKS_TABLE,
+    CREATE_TENURE_BLOCKS_ON_BLOCKS_TRIGGER,
+    CREATE_INDEXES_4,
+    MIGRATE_GLOBALLY_ACCEPTED_BLOCKS_TO_TENURE_BLOCKS,
+    "INSERT INTO db_config (version) VALUES (4);",
```

Comment:
Can we make sure there's only one row in `db_config`?  How about:

```sqlite
DELETE FROM db_config;
INSERT INTO db_config (version) values (4);
```

Reply:
In other places we do:
```
UPDATE db_config SET version = ?1
```
but we do always retrieve the max any way:
```
SELECT MAX(version) FROM db_config
```

Reply:
(I was just looking at this in #5452)

Reply:
Yes, and we can do better in both places :)

Reply:
Done.